spec/features/consumer/shops_spec.rb

```diff
@@ -201,4 +201,3 @@ def click_link_and_ensure(link_text, check)
     end
   end
 end
-
```

Comment:
Ok great I'll rebase in future.
Yeah was purely to force a buildkite build. Agreed it's ugly!